### PR TITLE
Snyder l2252 land input 5 irr mgmt

### PR DESCRIPTION
**Pull request by abigailsnyder**
again, only error on check package locally is for a function I didn't write or call,  fill_exp_decay_extrapolate

### CODE REVIEW

File: R/zchunk_L2252.land_input_5_irr_mgmt.R [hunk 1]
```diff
@@ -1,34 +1,51 @@
 #' module_aglu_L2252.land_input_5_irr_mgmt
 #'
-#' Briefly describe what this chunk does.
+#' This chunk produces L2252.LN5_Logit, L2252.LN5_HistMgdAllocation_crop, L2252.LN5_MgdAllocation_crop,
```
Comment (kvcalvin) [inline R/zchunk_L2252.land_input_5_irr_mgmt.R]: Maybe a little English too..."This chunk produces the inputs for the lowest level of the land nest, including disaggregated crop technologies."

File: R/zchunk_L2252.land_input_5_irr_mgmt.R [hunk 2]
```diff
@@ -50,141 +67,323 @@ module_aglu_L2252.land_input_5_irr_mgmt_DISABLED <- function(command, ...) {
...
+        left_join_error_no_match(GCAM_region_names, by = "GCAM_region_ID") %>%
+        replace_GLU(map = basin_to_country_mapping) ->
+        L181.YieldMult_R_bio_GLU_irr
+    }
+
+    # convert_LN4_to_LN5
+    # A function to carry LN4 information down to LN5
+    convert_LN4_to_LN5 <- function(data, names){
+      data %>%
+        repeat_add_columns(tibble::tibble(level = c( "lo", "hi" ))) %>%
```
Comment (bpbond) [inline R/zchunk_L2252.land_input_5_irr_mgmt.R]:
```
        repeat_add_columns(tibble(level = c("lo", "hi"))) %>%
```

File: R/zchunk_L2252.land_input_5_irr_mgmt.R [hunk 3]
```diff
@@ -50,141 +67,323 @@ module_aglu_L2252.land_input_5_irr_mgmt_DISABLED <- function(command, ...) {
...
+        mutate(LandNode5 = LandLeaf,
+               LandLeaf = paste(LandNode5, level, sep = aglu.MGMT_DELIMITER)) ->
+        data_new
+      data_new <- data_new[names]
+
+      return( data_new )
```
Comment (bpbond) [inline R/zchunk_L2252.land_input_5_irr_mgmt.R]: No GCAM-style parenthesis spacing please

File: R/zchunk_L2252.land_input_5_irr_mgmt.R [hunk 4]
```diff
@@ -50,141 +67,323 @@ module_aglu_L2252.land_input_5_irr_mgmt_DISABLED <- function(command, ...) {
...
+
+      # give land a corresponding id and filter to the land ids that
+      # occur in prod1
+      land %>%
+        mutate(id = paste0(region, LandLeaf, year)) %>%
+        semi_join(prod1, by = "id") %>%
+        select(-id) ->
+        land1
+
+      return(land1)
```
Comment (bpbond) [inline R/zchunk_L2252.land_input_5_irr_mgmt.R]: Can get of these last three lines and just end with `select(-id)`

**Comment by abigailsnyder** [comment]
corrections done

**Pull request merged**